remote/bitbucketserver/bitbucketserver.go

```diff
@@ -111,7 +105,7 @@ func (bs *BitbucketServer) Login(res http.ResponseWriter, req *http.Request) (*m
 	bits, err := ioutil.ReadAll(response.Body)
 	userName := string(bits)
 
-	response1, err := client.Get(bs.URL + "/rest/api/1.0/users/" +userName)
+	response1, err := client.Get(bs.URL + "/rest/api/1.0/users/" + userName)
```

Comment:
`fmt.Sprintf("%s/rest/api/1.0/users/%s", bs.URL, userName)` for string interpolation